lib/db_utils.pm

```diff
@@ -153,20 +155,22 @@ sub push_image_data_to_db {
     $args{arch} //= get_required_var('ARCH');
     $args{flavor} //= get_required_var('FLAVOR');
     $args{build} //= get_required_var('BUILD');
-    $args{type} //= '';
+    $args{table} //= 'size';
+    $args{url} = $job_url;
+    $args{asset} = $image;
+    $args{value} = $value;
+    $args{product} = $product;
     (my $build = $args{build}) =~ s/\_.*//;    #To remove unneeded strings, e.g. 15.11_init-image -> 15.11
 
-    my $cmd = 'curl -i -K /etc/postgres_conf -X POST http://' . $db_ip . ':' . $db_port . '/size -d \'{';
-    $cmd .= '"product": "' . $product . '", ';
-    $cmd .= '"distri": "' . $args{distri} . '", ';
-    $cmd .= '"version": "' . $args{version} . '", ';
-    $cmd .= '"arch": "' . $args{arch} . '", ';
-    $cmd .= '"flavor": "' . $args{flavor} . '", ';
-    $cmd .= '"build": "' . $build . '", ';
-    $cmd .= '"asset": "' . $image . '", ';
-    $cmd .= '"url": "' . $job_url . '", ';
-    $cmd .= '"type": "' . $args{type} . '", ';
-    $cmd .= '"value": "' . $value . '"}\'';
+    use Data::Dumper;
+    print("TESTING ...." . Dumper(%args));
+    assert_script_run('cat /etc/postgres_conf');
+    my $cmd = sprintf('curl -i -K /etc/postgres_conf -X POST http://%s:%s/%s', $db_ip, $db_port, $args{table});
+    delete $args{table};
+    $cmd .= " -d '";
+    $cmd .= encode_json(\%args);
+    $cmd .= q(');
```

Comment:
Maybe now is more readable put that in one single line
`$cmd .= " -d '" . encode_json(\%args) . q(');`

Reply:
oh, I have forgotten to remove the `Data::Dumper` debug lines xD
Yeah, well I have kinda kept the same logic that one line == one action. :) 
However, I do not mind to write it as you suggest :)